knack/invocation.py

```diff
@@ -180,9 +184,13 @@ def execute(self, args):
             del deprecate_kwargs['_get_message']
             deprecations.append(ImplicitDeprecated(**deprecate_kwargs))
 
+        # TODO: search for implicit preview?
```

Comment:
Please clarify a little, what may constitute an implicit preview?

Reply:
I see the class name below, so I'm guessing you mean differentiating behavior. However, I'm still not sure what would make something implicitly a preview. :)

Reply:
Actually, I do need to pull this code in from the CLI. You do need to have "implicit preview" so that if DLS is in preview, all of the subgroups and commands under it also recognize they are in preview.

Reply:
Got it, may be worth adding that to a doc comment somewhere. :)